src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAllele.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.Objects;
+
+/**
+ * This class represents the allele of an SV breakpoint (a novel adjacency between two genomic locations
+ */
+class BreakpointAllele {
+    final SimpleInterval leftAlignedLeftBreakpoint;
+    final SimpleInterval leftAlignedRightBreakpoint;
+    final String insertedSequence;
+    final String homology;
+    final boolean fiveToThree;
+    final boolean threeToFive;
+
+    public BreakpointAllele(final SimpleInterval leftAlignedLeftBreakpoint, final SimpleInterval leftAlignedRightBreakpoint, final String insertedSequence, final String homology, final boolean fiveToThree, final boolean threeToFive) {
+        this.leftAlignedLeftBreakpoint = leftAlignedLeftBreakpoint;
+        this.leftAlignedRightBreakpoint = leftAlignedRightBreakpoint;
+        this.insertedSequence = insertedSequence;
+        this.homology = homology;
+        this.fiveToThree = fiveToThree;
+        this.threeToFive = threeToFive;
+    }
+
+    public enum InversionType{
+        INV_3_TO_5, INV_5_TO_3, INV_UNKNOWN
```

Comment:
Just trying to guard/point out a weird state. I can change it to INV_NONE, I guess, since it shouldn't really exist.